lib/test_prof/factory_prof/printers/simple.rb

```diff
@@ -10,10 +10,18 @@ def dump(result)
           return log(:info, "No factories detected") if result.raw_stats == {}
           msgs = []
 
+          total = result.stats.inject(0) { |acc, stat| acc += stat[:total] }
```

Comment:
We could use `sum` here:

```suggestion
          total = result.stats.sum { |stat| stat[:total] }
```